### ISSUE TITLE
Error during minification

### ISSUE THREAD

**Issue by sirenkovladd** [putoutjs/minify#23]
- **Version** (`npm -v`): 10.5.2
- **Node Version** `node -v`: v20.13.1
- **OS** (`uname -a` on Linux): Darwin arm64

Error with redeclare value

```js
import { minify } from "@putout/minify";

const source = `function runUpdates() {
  let t = 1;
  1 ? (t = 0) : 1;
}
console.log(type, runUpdates)`;

let type = 'source'
eval(source);
const result = minify(source);
console.log(result);
type = 'minified'
eval(result);
```

```sh
❯ node i.js
source [Function: runUpdates]
...
TypeError: Property consequent of ConditionalExpression expected node to be of a type ["Expression"] but instead got undefined
loc: { line: 3, column: 7 },
  rule: 'remove-unreferenced-variables'
```

**Comment by coderaiser** [comment]
Landed in [`v3.19.4`](https://github.com/putoutjs/minify/releases/tag/v3.19.4) 🎉

**Comment by sirenkovladd** [comment]
thank you
